maint/code-cleanup.sh

```diff
@@ -91,6 +91,7 @@ all=0
 recursive=0
 got_file=0
 debug=
+filetype_list="\.c$|\.h$|\.c\.in$|\.h\.in$|\.cpp$|\.cpp.in$"
 ignore=0
 ignore_list="doc/|src/mpid/ch3/doc|src/mpid/ch3/include|src/mpid/ch3/src|src/mpid/ch3/util"
 ignore_list="$ignore_list|src/mpid/ch3/channels/nemesis/include|src/mpid/ch3/channels/nemesis/src|src/mpid/ch3/channels/nemesis/utils"
```

Comment:
How about append to ignore_list one file per line? It is difficult to read otherwise.

Reply:
Not really what I was aiming for in this PR. I’m sure there are other things that can be improved.